### PR TITLE
Feat: Added VideoCard and Navbar

### PR DESCRIPTION
**Pull request by spraveenofficial**
Added Navbar.
Added VideoCard.

### CODE REVIEW

File: frontend/src/Components/VideoCard/index.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import { useEffect } from "react";
+import "./style.css";
+import { fromEvent, of } from "rxjs";
+import { delay, takeUntil, flatMap } from "rxjs/operators";
+const VideoCard = () => {
+  useEffect(() => {
+    const card = document.querySelectorAll(".clip");
+    card.forEach((eachCard) => {
+      console.log(eachCard);
```
Comment (AmishMishra11) [inline frontend/src/Components/VideoCard/index.jsx:9]: remove console log before merging

File: frontend/src/Pages/Home/index.jsx [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
 const Home = () => {
   return (
-    <div>
-      <h1>Home</h1>
+    <div className="wrapper">
+      <VideoCard />
+      <VideoCard />
+      <VideoCard />
+      <VideoCard />
+      <VideoCard />
```
Comment (AmishMishra11) [inline frontend/src/Pages/Home/index.jsx:9]: You could use loop or map here

**Review by tanishk15sharma — APPROVED** [review]
good to merge

**Review by AmishMishra11 — APPROVED** [review]
You have followed good coding practices. You can merge it just after you have made some small suggested changes.

**Pull request merged**